### DISCUSSION TITLE
Non-AWS architectures?

### DISCUSSION THREAD

**Discussion by davidjeba** [brefphp/bref#1183] [General]
any possibilities on Google Firebase and Cloudflare Workers as platforms?

**Comment by mnapoli** [comment]
No, from the documentation:

> The choice of AWS as serverless provider is deliberate: at the moment AWS is the leading hosting provider, it is ahead in the serverless space in terms of features, performance and reliability.

**Comment by t-richard** [comment]
For Google cloud, you may want to take a look at https://github.com/php-runtime/google-cloud

For cloudflare workers, as far as I know you can't have custom runtimes and php is not supported out of the box. IIRC cloudflare workers are tied to the v8 engine so only js and web assembly are supported (not 100% sure).
